### ISSUE TITLE
[feature request] export as zone file

### ISSUE THREAD

**Issue by a1d3s** [ngoduykhanh/PowerDNS-Admin#836]
Sometimes i need an export from an zone , therefor i use at this time the print function from browser.
i think it would be a better solution if i can export the zone to an txt file or somethink else

**Comment by florianbeer** [comment]
You could connect to your pdns server via SSH and use `pdnsutil list-zone $DOMAIN` to get a textual representation of any zone.

**Comment by a1d3s** [comment]
@florianbeer yes i know , but if i use a frontend then it would be fine to have this feature here.
i'm not using it alone , but i'm the only one with access to ssh ;)

**Comment by jbe-dw** [comment]
Duplicate of #528
